docker/Dockerfile

```diff
@@ -21,6 +21,7 @@ RUN apt-get update \
 		curl \
 		# Used by binutils in the build process
 		flex \
+		libfl2 \
```

Comment:
Please add a comment why this dependency is needed: `# Required by flex but not a hard dependency in Debian.`